util/util.js

```diff
@@ -160,6 +161,12 @@ function generateStrictDateTs(ts) {
   }
 }
 
+function isHashDomain(domain) {
```

Comment:
H3dF2wLoc6Br1d89gtM2vV6DK215zY.XUB8JoptqZZ8=
Will the hash pass the test if it contains "."?

Reply:
Also make sure clear text domain will not match this